controller/src/beerocks/bml/internal/bml_internal.cpp

```diff
@@ -1852,6 +1852,53 @@ int bml_internal::register_nw_map_update_cb(BML_NW_MAP_QUERY_CB pCB)
     return (BML_RET_OK);
 }
 
+int bml_internal::device_oper_radios_query(BML_DEVICE_DATA *device_data)
+{
+    // Initialize the promise for receiving the response
+    beerocks::promise<bool> prmDeviceDataGet;
+    m_prmDeviceDataGet = &prmDeviceDataGet;
+    int iOpTimeout     = RESPONSE_TIMEOUT; // Default timeout
```

Comment:
Don't use cample case for simple variables, its against the coding style.
```suggestion
    int timeout = RESPONSE_TIMEOUT; // Default timeout
```
(nitpick) since it is already like that in this file..